project/src/test/java/com/google/sps/data/PlacesFetcherTest.java

```diff
@@ -109,15 +124,19 @@ private static URL createTestURL(String s) {
   }
 
   private static PlaceDetails createTestPlaceDetails(
-        String name, URL url, String phone, float rating, PriceLevel priceLevel, LatLng location) {
+        String name, URL website, String phone, float rating,
+        PriceLevel priceLevel, LatLng location, URL url, String id, String status) {
     PlaceDetails placeDetails = new PlaceDetails();
     placeDetails.name = name;
-    placeDetails.website = url;
+    placeDetails.website = website;
     placeDetails.formattedPhoneNumber = phone;
     placeDetails.rating = rating;
     placeDetails.priceLevel = priceLevel;
     placeDetails.geometry = new Geometry();
     placeDetails.geometry.location = location;
+    placeDetails.url = url;
```

Comment:
If this is the google URL maybe it should be clearer (adding "google")?

Reply:
Those are the fields of the google places API (of the PlaceDetails type), I agree it is very confusing so I changed the function parameter like you suggested.